### PR TITLE
Tenant QE account

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [ ] enhancement
- [x] documentation
- [ ] dependencies

## Description
Add tenant:
- Eyal qe account

Used the cloud-governance es_host and es_port.

### CODE REVIEW

**Review by ebattat — COMMENTED** [review]
Lets review it together !!!

File: jenkins/tenant/aws/qe/TaggingJenkinsfileHourly [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    stages {
+        stage('Checkout') { // Checkout (git clone ...) the projects repository
+           steps {
+                 checkout scm
+           }
+        }
+        stage('Initial Cleanup') {
+            steps {
+                 sh '''if [[ "$(podman images -q quay.io/ebattat/cloud-governance 2> /dev/null)" != "" ]]; then podman rmi -f $(podman images -q quay.io/ebattat/cloud-governance 2> /dev/null); fi'''
+            }
```
Comment (ebattat) [inline jenkins/tenant/aws/qe/TaggingJenkinsfileHourly:27]: Why are we still using quay.io/ebattat/cloud-governance  ?

File: jenkins/tenant/aws/qe/PolicyJenkinsfileDaily [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    }
+    stages {
+        stage('Checkout') { // Checkout (git clone ...) the projects repository
+           steps {
+                 checkout scm
+           }
+        }
+        stage('Initial Cleanup') {
+            steps {
+                 sh '''if [[ "$(podman images -q quay.io/ebattat/cloud-governance 2> /dev/null)" != "" ]]; then podman rmi -f $(podman images -q quay.io/ebattat/cloud-governance 2> /dev/null); fi'''
```
Comment (ebattat) [inline jenkins/tenant/aws/qe/PolicyJenkinsfileDaily:35]: Why are we still using quay.io/ebattat/cloud-governance ?

File: jenkins/tenant/aws/qe/PolicyJenkinsfileDaily [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                        sh 'python3 jenkins/tenant/aws/common/run_policies.py'
+                                        }
+                    }
+                }
+            }
+        }
+        stage('Finalize Cleanup') {
+            steps {
+                 sh '''if [[ "$(podman images -q quay.io/ebattat/cloud-governance 2> /dev/null)" != "" ]]; then podman rmi -f $(podman images -q quay.io/ebattat/cloud-governance 2> /dev/null); fi'''
+                 deleteDir()
```
Comment (ebattat) [inline jenkins/tenant/aws/qe/PolicyJenkinsfileDaily:72]: Why are we still using quay.io/ebattat/cloud-governance ?

File: jenkins/tenant/aws/qe/TaggingJenkinsfileHourly [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                        sh 'python3 jenkins/tenant/aws/common/run_tagging.py'
+                                        }
+                    }
+                }
+            }
+        }
+        stage('Finalize Cleanup') {
+            steps {
+                 sh '''if [[ "$(podman images -q quay.io/ebattat/cloud-governance 2> /dev/null)" != "" ]]; then podman rmi -f $(podman images -q quay.io/ebattat/cloud-governance 2> /dev/null); fi'''
+                 deleteDir()
```
Comment (ebattat) [inline jenkins/tenant/aws/qe/TaggingJenkinsfileHourly:46]: Why are we still using quay.io/ebattat/cloud-governance ?

**Review by ebattat — COMMENTED** [review]
Pls fix quay image in all the places:
quay.io/ebattat/cloud-governance => quay.io/cloud-governance/cloud-governance

**Review by ebattat — CHANGES_REQUESTED** [review]
Open PR to fix all the Jenkinsfile
quay.io/ebattat/cloud-governance => 
quay.io/cloud-governance/cloud-governance

`QUAY_CLOUD_GOVERNANCE_REPOSITORY=quay.io/cloud-governance/cloud-governance`

**Comment by athiruma** [comment]
> Fix as following: quay.io/ebattat/cloud-governance => quay.io/cloud-governance/cloud-governance
> 
> `QUAY_CLOUD_GOVERNANCE_REPOSITORY=quay.io/cloud-governance/cloud-governance`

Yup done.

File: jenkins/tenant/aws/qe/TaggingJenkinsfileHourly (hunk 4 above)
Comment (athiruma) [reply jenkins/tenant/aws/qe/TaggingJenkinsfileHourly:47]: I opened this PR, before we merging the change

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**